[documented-lasagna]: Updated instructions. (#603)
exercism/julia: exercises/concept.wip/documented-lasagna/.docs/instructions.md

# Instructions

In this exercise you're going to document the code you just wrote to help you cook a brilliant lasagna.

> You want to be able to go back to your code and remember what each function does and how to use it.
> You should describe the content of the function and relevant information, this should be written as a docstring.
> 

Reviewer: ```suggestion
You are unlikely to look at this lasagna-related code ever again, but documenting code is really helpful if you want any larger project to be understandable to others (or to yourself)!

To document your code, write a "docstring" for each of the three functions. Here's how to do it:

- Review the [Documentation][doc] section in the Julia Manual.
- Start with the function signature indented with 4 spaces.
- Include a single one-line sentence describing what the function does, in your own words.
```

~~~~exercism/note
You're provided with the exemplar solution of the [previous exercise](https://exercism.lol/tracks/julia/exercises/lasagna).
You may also choose your own solution to that exercise as a starting point if you prefer.
~~~~

## 1. Document preptime

The first function you wrote was `preptime`, which calculates how long it takes to prepare the lasagna.

## 2. Document remaining_time

The second function you wrote was `remaining_time`, which calculates how long it is until the lasagna is ready.

## 3. Document total_working_time

> The third function you wrote was `total_working_time`, which calculates how long it takes to make the lasagna.

Reviewer: ```suggestion
The third function you wrote was `total_working_time`, which calculates how long it takes to make the lasagna.

[doc]: https://docs.julialang.org/en/v1/manual/documentation/
```

# Instructions

In this exercise you're going to document the code you just wrote to help you cook a brilliant lasagna.
You are unlikely to look at this lasagna-related code ever again, but documenting code is really helpful if you want any larger project to be understandable to others (or to yourself)!

To document your code, write a "docstring" for each of the three functions. Here's how to do it:

- Review the [Documentation][doc] section in the Julia Manual.
- Start with the function signature indented with 4 spaces.

> - Include a single one-line sentence describing what the function does, in your own words.

Reviewer: ```suggestion
- Include a single one-line sentence describing what the function does, in your own words.

```
